code/src/Album.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import './components/album.css';
+import './components/icons.css';
+/* import data from './data.json'*/
+
+/*console.log(data)
+console.log(data.album.items[1])*/
+
+export const Album = (props) => {
+    return (
```

Comment:
Looks like your editor is set up to do 4 spaces for indentation (should be 2). See [this question on stack overflow](https://stackoverflow.com/c/technigo/questions/253) for a bunch of things to check.